package.json

```diff
@@ -40,6 +40,7 @@
     "chalk": "^3.0.0",
     "conf": "^6.1.0",
     "execa": "^4.0.0",
+    "fuzzy": "^0.1.3",
```

Comment:
Can we replace this dependency for something else that is more maintained ?

https://www.npmjs.com/package/fuse.js